scraper.go

```diff
@@ -106,12 +107,13 @@ type logglyJsonMsg struct {
 	JavaClass        string  `json:"java_class,omitempty"`
 }
 
-func (m *LogMsg) toLogglyJson(hostname string, source string, target *json.Encoder) error {
+func (m *LogMsg) toLogglyJson(hostname string, OwnHostname string, source string, target *json.Encoder) error {
```

Comment:
All of our other JSON fields are lower_case, so stick to that for the JSON message.